packages/client/src/v2-events/features/events/actions/print-certificate/pdfUtils.ts

```diff
@@ -144,6 +145,16 @@ export function compileSvg({
     }
   )
 
+  Handlebars.registerHelper(
+    'findUserById',
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    function (this: any, u: User[], id: string) {
```

Comment:
I think we can do:

```
  Handlebars.registerHelper(
    'formatDate',
    (_, dateString: string, formatString: string) => {
      const date = new Date(dateString)
      return isValid(date) ? format(date, formatString) : ''
    }
  )
```

Though doesn't matter much.

Handlebars [should](https://handlebarsjs.com/guide/installation/) also be imported as: `import Handlebars from "handlebars";`, but also doesn't matter much.